stellar_sdk/soroban_server.py

```diff
@@ -236,6 +237,37 @@ def send_transaction(
         )
         return self._post(request, SendTransactionResponse)
 
+    def poll_transaction(self, transaction_hash: str, attempts: int = 30, sleep_strategy: int = 1) -> GetTransactionResponse:
+        """Poll for a particular transaction with certain parameters.
+
+        After submitting a transaction, clients can use this to poll for transaction completion and return a definitive state of success or failure.
+
+        :param transaction_hash: The hash of the transaction to poll for.
+        :type transaction_hash: str
+        :param attempts: The number of attempts to make before returning the last-seen status, defaults to 30.
+        :type attempts: int, optional
+        :param sleep_strategy: The amount of time, in seconds, to wait for between each attempt, defaults to 1.
```

Comment:
Nit. Should we set this to a higher value? The lower bound is the ledger closing time right? I would put 2 as to limit a little bit the impact on RPC calls.

Reply:
@overcat Any thoughts on the default length for the "basic" sleep strategy?